src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -255,7 +255,7 @@ public async Task<Document> ApplyCodeFixesForSpecificDiagnosticIdAsync(Document
             var fixAllService = document.Project.Solution.Workspace.Services.GetRequiredService<IFixAllGetFixesService>();
 
             var solution = await fixAllService.GetFixAllChangedSolutionAsync(
-                fixCollection.FixAllState.CreateFixAllContext(progressTracker, cancellationToken)).ConfigureAwait(false);
+                new FixAllContext(fixCollection.FixAllState, progressTracker, cancellationToken)).ConfigureAwait(false);
```

Comment:
`CreateFixAllContext` was inlined here as the `FixAllState.cs` has been moved to the shared layer, but creating a new FixAllContext is only required from IDE code fix service engine and test layer.